docs/examples/example_survival.ipynb

```diff
@@ -0,0 +1,206 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "from git_root import git_root\n",
+    "\n",
+    "rossi = pd.read_csv(git_root(\"data/rossi.csv\"))\n",
+    "rossi.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Next we will create the ``X``, ``y`` and ``w`` variables necessary for training our MetaLearner. In this dataset we have 6 feature columns but we will also add a column ``\"censored\"`` to ``X`` which indicates if the observation was censored (meaning that the event did not occur before a time limit -- one year in this case)."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "feature_columns = [\"age\", \"race\", \"wexp\", \"mar\", \"paro\", \"prio\"]\n",
+    "X = rossi[feature_columns].copy(deep=True)\n",
+    "X[\"censored\"] = 1 - rossi[\"arrest\"]\n",
+    "for c in [\"race\", \"wexp\", \"mar\", \"paro\", \"censored\"]:\n",
+    "    X[c] = X[c].astype(\"category\")\n",
+    "\n",
+    "y = rossi[\"week\"]\n",
+    "w = rossi[\"fin\"]"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Creating the base models\n",
+    "\n",
+    "Some survival models face two primary challenges:\n",
+    "\n",
+    "* **Time to event prediction:** Certain models, particularly those relying on the [Cox Proportional Hazards Model](https://en.wikipedia.org/wiki/Proportional_hazards_model), may not directly provide the time to event prediction. Instead, they yield hazard proportions; however, the desired output is the more useful and direct measure of time to the event.\n",
+    "\n",
+    "* **Censoring Support:** Survival analysis involves dealing with censored data, where the outcome variable ``y`` may not always be a singular real number for each observation and could be an interval instead.\n",
+    "\n",
+    "\n",
+    "To address these issues, we need to construct a model to act as the nuisance model. Here's how we can address the associated problems:\n",
+    "\n",
+    "* **Time to event prediction:** The principle to remember is that this nuisance model, when invoking the ``predict`` method, must return a time to event prediction for each observation. In this case, we will employ [``xgboost`` with Accelerated Failure Time](https://xgboost.readthedocs.io/en/stable/tutorials/aft_survival_analysis.html), since its output is inherently the time to event. However, other models may necessitate integration over the survival function or necessitate other transformations.\n",
```

Comment:
nit:
```suggestion
    "* **Time to event prediction:** The principle to remember is that this nuisance model, when invoking the ``predict`` method, must return a time to event prediction for each observation. In this case, we will employ [``xgboost`` with Accelerated Failure Time](https://xgboost.readthedocs.io/en/stable/tutorials/aft_survival_analysis.html), since its output is inherently the time to event. However, other models may necessitate integration over the survival function or require other transformations to get expected survival time predictions.\n",
```